### ISSUE TITLE
parse issue with doubles at repl?

### ISSUE THREAD

**Issue by alrunner4** [edwinb/Idris2-boot#171]
# Steps to Reproduce
```
~$ idris2
     ____    __     _         ___                                           
    /  _/___/ /____(_)____   |__ \                                          
    / // __  / ___/ / ___/   __/ /     Version 0.0.0-ab98b4d3c
  _/ // /_/ / /  / (__  )   / __/      http://www.idris-lang.org            
 /___/\__,_/_/  /_/____/   /____/                                           

Welcome to Idris 2.  Enjoy yourself!
Main> 2.0 - 2.0
4
```

# Expected Behavior
```
Main> 2.0 - 2.0
0
```

# Observed Behavior
Subtraction on doubles does... addition?

**Comment by christian-public** [comment]
Good catch!

It seems like the problem only affects the interpreter. `:exec printLn (2.0 - 2.0)` works as expected.

Fix is available in #174.

**Comment by edwinb** [comment]
Oh, that's an entertaining one! Thanks for the fix @chrrasmussen, I'll catch up with PRs when I'm back at work...
